src/solution.c

```diff
@@ -102,9 +103,30 @@ void solution_send_nmea(gnss_solution *soln, dops_t *dops,
   );
 }
 
+/** Creates and sends RTK solution.
+ * If the base station position is known,
+ * send the NMEA and SBP psuedo absolute msgs.  
+ *
+ * \note this function relies upon the global base_pos_ecef and base_pos_known
+ * for logic and base station position when sending psuedo absolutes.
+ * If operating in simulation mode, it depends upon the simulation mode enabled 
+ * and the simulation base_ecef position (both available in the global struct
+ * sim_settings and accessed via wrappers prototyped in simulator.h)
+ * 
+ * \param *t pointer to gps time struct representing gps time for solution
```

Comment:
I don't think Doxygen wants the `*` here, at least in the rest of the docs we don't put the pointer qualifier, just the name